lib/jsonwp-proxy/proxy.js

```diff
@@ -26,15 +26,15 @@ class JWProxy {
   }
 
   endpointRequiresSessionId (endpoint) {
-    return !_.includes(['/session', '/sessions', '/status'], endpoint);
+    return !_.includes(['/session', '/sessions', '/status', '/homescreen'], endpoint);
```

Comment:
It doesn't require a sesion.  With Facebook's WDA you don't need a session to execute this command.  Also with Facebook's WDA you can automate part of simulator outside of your application.  

I could send a pull request to Facebook's WDA and see if we can also add `homescreen` as a session command if that would work better.